src/types/Privacy.ts

```diff
@@ -83,4 +83,24 @@ export type PrivacyType = {
      * @note The debug geography is used only when the app is in debug mode.
      */
     setConsentFlowDebugUserGeography(userGeography: ConsentFlowUserGeography): void;
+
+    /**********************************************************************************/
+    /* Google-certified CMP */
+    /**********************************************************************************/
+
+    /**
+     * Shows the CMP flow to an existing user.
+     * Note that the user's current consent will be reset before the CMP alert is shown.
+     *
+     * The function returns when the flow finishes showing.  On success, returns nothing.  On
+     * failure, returns one of the CmpError code.
+     *
+     * @return {Promise<CmpError|null>}
+     */
+    showCmpForExistingUser(): Promise<CmpError | null>;
+
+    /**
+     * @return {true} if a supported CMP is integrated.
+     */
```

Comment:
Let's copy docs from Unity so we're consistent there: https://github.com/AppLovin/Unity-Plugin/blob/78fec81f5e7b1e0f212d439fa4ff8f7b2ab689e8/Plugin/Assets/MaxSdk/Scripts/MaxCmpService.cs#L71C15-L71C70